test/common/libs/getDebuffPotionItems.test.js

```diff
@@ -0,0 +1,51 @@
+
+import {
+  generateUser,
+} from '../../helpers/common.helper';
+
+import getDebuffPotionItems from '../../../website/common/script/libs/getDebuffPotionItems';
+import { TRANSFORMATION_DEBUFFS_LIST } from '../../../website/common/script/constants';
+
+describe('getDebuffPotionItems', () => {
+  let user;
+
+  beforeEach(() => {
+    user = generateUser();
+  });
+
+  for (const key in TRANSFORMATION_DEBUFFS_LIST) {
```

Comment:
For this one an easier way to write it that doesn't require the Object.prototype check is to use `for (const key in Object.keys(TRANSFORMATION_DEBUFFS_LIST))`

Reply:
It is still push me to wrap code in hasOwnProperty check due to eslint rule _guard-for-in_. If I change it to 
`for (const key in Object.keys(TRANSFORMATION_DEBUFFS_LIST))` the situation still the same... As well I search in project and see that it will be first place for `in Object.keys(`. 
Are you sure that it will be better that current one?

Reply:
Sorry, it should be `for (const key of Object.keys(TRANSFORMATION_DEBUFFS_LIST))`

Reply:
Yeah, but it means I mainly use values, but not keys which I need in this loop. I think it is good option for arrays where we don't need index, but not so good in this keys where I need key and value.

Reply:
Hmm if you do a for...of on Object.keys you’re going to get the keys. You can also get values using Object.values() or both with Object.entries()

There are a few examples of using for of with Object.keys in the codebase

Reply:
Yes, you are right, sorry... Dat night coding :)